.github/workflows/ci-test.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
           - 8545:8545
           - 8546:8546
         env:
-          OASIS_DEPOSIT: /oasis-deposit -test-mnemonic -n 5
+          OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
```

Comment:
Good catch, I think we should change behavior of `sapphire-localnet` to use deterministic addresses by default, requiring a different parameter to randomize the accounts.

Nice work, getting CI working.

Note that the `/CONTAINER_READY` file is now being populated in the latest `sapphire-localnet`, this moves all the success checks into the container so we only have to check if a single file exists.

https://github.com/oasisprotocol/sapphire-paratime/blob/0c370b6e423313a72866ed1cc7b2188122388887/.github/workflows/contracts-test.yaml#L22